data/summary/types/summary.go

```diff
@@ -73,16 +80,11 @@ type DeviceDataCursor interface {
 	Close(ctx context.Context) error
 }
 
-type UserLastUpdated struct {
-	FirstData time.Time
-	LastData  time.Time
-
-	EarliestModified time.Time
-
-	LastUpload time.Time
-
-	LastUpdated     time.Time
-	NextLastUpdated time.Time
+type DeviceDataManager interface {
```

Comment:
Consider renaming this - "manage" implies mutating the data, but this interface only queries the data store. Is this a `DataFetcher` for summary data?

Reply:
Thats a better name, changed.